Helper/Locale.php

```diff
@@ -2,64 +2,49 @@
 
 namespace Lipscore\RatingsReviews\Helper;
 
-use Lipscore\RatingsReviews\Helper\AbstractHelper;
-
 class Locale extends AbstractHelper
 {
-    protected static $availableLocales = [
+    public const AVAILABLE_LOCALES = [
         'en', 'it', 'no', 'es', 'br', 'ru', 'se', 'cz', 'nl', 'dk', 'ja', 'de', 'fi', 'fr'
```

Comment:
Could you update the list of supported languages? The current list is in invitation[lang] section of the documentation: https://members.lipscore.com/apidoc/v2/purchases/create.en.html

Reply:
I think it's out of this PR scope. We should do it but as another update on both extension versions: 1.x and 2.x(the current new widget implementation)